requirements.txt

```diff
@@ -1,6 +1,6 @@
 csvvalidator==1.2
 jsonschema==2.5.1
--e git+https://github.com/openregister/conformance-test.git@7c837fd7f4ceabd1f2a3521e76504c4af48c3112#egg=openregister_conformance-register-resource
+-e git+https://github.com/openregister/conformance-test.git@0014da294c91b652932e85257819a310dd932f51#egg=openregister_conformance-register-resource
```

Comment:
Conformance tests will start pointing to master once the conformance PR  https://github.com/openregister/conformance-test/pull/14 is merged